inc/define.php

```diff
@@ -317,7 +317,7 @@
 
 $CFG_GLPI['itemdevicesimcard_types']      = ['Computer', 'Peripheral', 'Phone', 'NetworkEquipment', 'Printer'];
 
-$CFG_GLPI['itemdevicegeneric_types']      = ['*'];
+$CFG_GLPI['itemdevicegeneric_types']      = ['Computer', 'Peripheral', 'Phone', 'NetworkEquipment', 'Printer'];
 
 $CFG_GLPI['itemdevicepci_types']          = ['*'];
```

Comment:
Maybe the best way to enable all GLPI itemtypes that are compatible to devices is to do this:
```suggestion
$CFG_GLPI['itemdevicegeneric_types']      = $CFG_GLPI["itemdevices_types"];

$CFG_GLPI['itemdevicepci_types']          = $CFG_GLPI["itemdevices_types"];
```